web/src/lib/components/asset-viewer/activity-viewer.svelte

```diff
@@ -109,7 +112,7 @@
   const handleDeleteReaction = async (reaction: ActivityResponseDto, index: number) => {
     try {
       await deleteActivity({ id: reaction.id });
-      reactions.splice(index, 1);
+      activityManager.activities = activityManager.activities.splice(index, 1);
```

Comment:
should this not be like "removeReaction(id)" or something. I would maybe even put the `deleteActivity()` call in the manager itself.